gcloud-java-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1213,6 +1213,12 @@ public static Builder builder() {
   /**
    * Creates a new bucket.
    *
+   * <p>Example of creating a bucket.
```

Comment:
No, these are the snippets in `StorageSnippets` after they are inserted into the javadoc using the `add_snippets_to_file.py` script. The reason for this is that I want the javadoc that we check in into the repository to be complete, not missing any part. Hence, running the `add_snippets_to_file.py` script is a pre-commit action.

This can be changed, but I would prefer to have complete javadoc in the source code for now.

I believe this also answers your following question.